### PR TITLE
Rebase completed

### PR DESCRIPTION
**Pull request by tulasikarnati**
all merge strategies are working

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -5,4 +5,9 @@ feature-1
 
 this is done by developers
 developer is doing some development
-developement completed
\ No newline at end of file
+developement completed
+
+### rebase
+-it is another merge strategy
+-second line of code
```
Comment (tgopic1991) [inline README.MD:12]: this line is okay?

**Review by tgopic1991 — COMMENTED** [review]
please check again

**Comment by tulasikarnati** [comment]
i did the alteration, please confirm

**Review by tgopic1991 — APPROVED** [review]
okay

**Pull request merged**